pkgs/development/python-modules/pslab/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, buildPythonPackage, fetchPypi, isPy27, numpy, pyserial, scipy }:
+
+buildPythonPackage rec {
+  pname = "pslab";
+  version = "2.4.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0iifjz7j6g982sc1id91d2krrv6zr42v2fzln9kw7nwjzg1j2j3a";
+  };
+
+  disabled = isPy27;
+
+  # Testing requires Pil, some patches...
```

Comment:
Please give it a try.

Reply:
I have no idea of these things, tbh

Reply:
I learnt that tests are not included in the pypi archive again. Noted.

Apparently all pypi archives I touched lacked tests. I didn't expect they trim tests out. When I was trying to run the tests I was always using a git clone of the repository and I didn't imagine the pypi archive being much different.